R/checks.R

```diff
@@ -789,4 +789,45 @@ check_fracwith_diswith <- function(form, rep) {
     filter(ptstatus == 1 & (num_of_fractures > 1 |
                               location_of_fracture != location_of_dislocation))
   return(problems)
+}
+
+#' Check that the time from injury to def. stabilization should be within
+#'  1 hour of the time from injury to abx administration
+#' 
+#' @param form dataframe containing ptstatus, form 4.1, and form5.3
+#' @return a dataframe containing problematic entries with relevant columns
+#' @import tidyverse
+#' @export 
+check_locof1stadmin_pritosur <- function(form) {
```

Comment:
Use names of columns instead of inventing your own name. If you must invent your own names, avoid non-standard abbreviations.

Reply:
How they column names? I can't find them.